i18n/en.json

```diff
@@ -1,9 +1,23 @@
 {
-	"@metadata": {
-		"authors": [
-			"Christian Dullweber",
-			"Katie Filbert"
-		]
-	},
-	"propertysuggester-desc": "Suggests properties when adding Wikibase statements."
+    "@metadata": {
+        "authors": [
+            "Christian Dullweber",
+            "Katie Filbert"
+        ]
+    },
+    "propertysuggester-desc": "Suggests properties when adding Wikibase statements.",
+    "apihelp-wbsgetsuggestions-description": "API module to get property suggestions for Wikibase entities.",
+    "apihelp-wbsgetsuggestions-example-1": "Get suggestions for entity Q4",
+    "apihelp-wbsgetsuggestions-example-2": "Get suggestions for entity Q4 from rank 10 to 15",
+    "apihelp-wbsgetsuggestions-example-3": "Get suggestions for the property combination P21 and P31",
+    "apihelp-wbsgetsuggestions-example-4": "Get suggestions for the qualifier that are used with P21",
+    "apihelp-wbsgetsuggestions-example-5": "Get suggestions for the references that are used with P21",
+    "apihelp-wbsgetsuggestions-param-context": "Either item, reference or qualifier.",
```

Comment:
This line does not add any knowledge. But well, I can see this was not created in this patch.